.gitignore

```diff
@@ -115,8 +115,11 @@ website/www/site/code_samples
 website/www/site/_config_branch_repo.toml
 website/www/yarn-error.log
 !website/www/site/content
+.test-infra/validate-runner/src/main/js/apache-beam-testing-keys.json
 
 # Node
+sdks/typescript/node_modules
+node_modules/**
```

Comment:
There are still multiple website dependency files in the repo root directory e.g. `package-locks.json`, `package.json`, `yarn.lock`. Should organize it in somewhere under `website` folder.

There is already a project root in "website/www/" , there is an `yarn.lock` and `package.json` file there for the website dependency. (and `website/www/node_modules` is already in .gitignore)

Or If it is necessary to create a different project to generete the capability matrix, could move them into `website/generate_capability_matrix` project or so. 

May ask @pabloem which one of the choice is preferred.